### ISSUE TITLE
remove erlpack from package.json

### ISSUE THREAD

**Issue by JamesPierce82** [MAD-Club/RinsedBot#24]
After attempting to get node running on my windows 10 laptop for a few hours, it appears that erlpack is the main culprit behind not being able to get it working after a myriad of other steps.

@j-nguyen  are we able to get by without using this?

**Comment by JamesPierce82** [comment]
I should specify that if you attempt to run it without erlpack, it fails on node-opus, once run with erlpack it fails on erlpack install, then you can remove erlpack to successfully run npm install... it's a very long process and it still logs some errors, however it WILL allow you to run a bot after stepping through adding and removing it. :/

**Comment by j-nguyen** [comment]
I believe erlpack and node-opus are a part of discordjs, so I don't think you can entirely remove them

**Comment by JamesPierce82** [comment]
yeah, it looks like node-opus is used to any voice content, and erlpack is one of the optional dependencies, but not one that looks like we should remove it.
Unfortunately, when I asked around in the discord.js server, I was pretty much told if its not working on windows to switch to linux, as all they really know is to try it with the MS Build tools, outside of that I wasn't getting much help.
